include/bcs/VacuumConcBC.h

```diff
@@ -22,9 +25,15 @@ class VacuumConcBC : public IntegratedBC, public ScalarTransportBase
 
 protected:
   virtual Real computeQpResidual() override;
-
   virtual Real computeQpJacobian() override;
 
+  enum BC_TYPE
+  {
+    MARSHAK,
```

Comment:
Should these be lowercase (to match below)? I'm not sure the style guide here. Just checking, because in NtAction, they're enumerated in lowercase.

Reply:
It's kind of weird. Most instances of MooseEnum in the MOOSE repo use lowercase in their declarations. But MooseEnum automatically converts entries to uppercase for `==` operations to make them "case-insensitive". For my switch case statements, I need to separately declare enums in uppercase because of the conversion.